arnold/plugins/gaffer.mtd

```diff
@@ -931,6 +961,90 @@
 		gaffer.plugType STRING ""
 
 
+[node car_paint]
+	
+	gaffer.nodeMenu.category STRING "Surface"
+
+	gaffer.graphEditorLayout.defaultVisibility BOOL false
+
+
+	[attr base]
+		gaffer.graphEditorLayout.visible BOOL true
+		page STRING "Base"
+	[attr base_color]
+		gaffer.graphEditorLayout.visible BOOL true
+		page STRING "Base"
+	[attr base_roughness]
+		page STRING "Base"
+		gaffer.graphEditorLayout.visible BOOL true
+
+	[attr specular]
+		gaffer.graphEditorLayout.visible BOOL true
+		page STRING "Specular"
+	[attr specular_color]
+		gaffer.graphEditorLayout.visible BOOL true
+		page STRING "Specular"
+	[attr specular_roughness]
+		gaffer.graphEditorLayout.visible BOOL true
+		page STRING "Specular"
+	[attr specular_IOR]
+		page STRING "Specular"
+
+	[attr specular_flip_flop]
+		page STRING "Specular"
+	[attr specular_light_facing]
+		page STRING "Specular"
+	[attr specular_falloff]
+		page STRING "Specular"
+
+	[attr transmission_color]
+		page STRING "Transmission"
+
+	[attr flake_color]
+		gaffer.graphEditorLayout.visible BOOL true
+		page STRING "Flake"
+	[attr flake_roughness]
+		gaffer.graphEditorLayout.visible BOOL true
+		page STRING "Flake"
+	[attr flake_IOR]
+		page STRING "Flake"
+
+	[attr flake_scale]
+		page STRING "Flake"
+	[attr flake_density]
+		page STRING "Flake"
+	[attr flake_layers]
+		page STRING "Flake"
+	[attr flake_normal_randomize]
+		page STRING "Flake"
+	[attr flake_coord_space]
+		page STRING "Flake"
+
```

Comment:
What do you think about adding `pref_name` here too, so it doesn't show up above everything else. I think it matches where they document it in the MtoA docs:

![Screenshot 2020-02-14 at 13 46 00](https://user-images.githubusercontent.com/896779/74536625-5cabaa00-4f30-11ea-9656-f99476a0454d.png)

If you need to adjust the ordering, I added support for it a little while back via: https://github.com/GafferHQ/gaffer/blob/f146aa8acee4f0c93b4bc595c788bf185831b6d0/arnold/plugins/gaffer.mtd#L46